pkg/autoscaler/config/config.go

```diff
@@ -176,6 +178,10 @@ func validate(lc *autoscalerconfig.Config) (*autoscalerconfig.Config, error) {
 		return nil, fmt.Errorf("initial-scale = %v, must be at least 0 (or at least 1 when allow-zero-initial-scale is false)", lc.InitialScale)
 	}
 
+	if lc.MinScale < 0 {
```

Comment:
probably worth also checking minScale is < maxScale

Reply:
added